pkg/plugins/common/kustomize/v1/init.go

```diff
@@ -43,10 +50,17 @@ type initSubcommand struct {
 }
 
 func (p *initSubcommand) UpdateMetadata(cliMeta plugin.CLIMetadata, subcmdMeta *plugin.SubcommandMetadata) {
-	subcmdMeta.Description = `Initialize a common project including the following files:
+	subcmdMeta.Description = fmt.Sprintf(`Initialize a common project including the following files:
   - a "PROJECT" file that stores project configuration
   - several YAML files for project deployment under the "config" directory
-`
+
+  NOTE: The kustomize/v1 plugin used to do this scaffold use its v3 release (%s).
+Therefore, darwin/arm64 is not supported since the project does not provide
```

Comment:
Just to be a bit more specific:
```suggestion
Therefore, darwin/arm64 is not supported since Kustomize does not provide v3
```